controller/utils.go

```diff
@@ -13,6 +16,104 @@ import (
 	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta2"
 )
 
+var (
+	maxBackoff      int64
+	maxBackoffMutex sync.RWMutex
+)
+
+type ExponentialBackoff struct {
```

Comment:
NIT: This and the methods seems like good candidate to move into [go-common-libs](https://github.com/longhorn/go-common-libs)